packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -391,8 +391,6 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         //   2. The amount of gas provided to the call to the target contract is at least the gas
         //      limit specified by the user. If there is not enough gas in the callframe to
         //      accomplish this, `callWithMinGas` will revert.
```

Comment:
Would the following be slightly more accurate?

```suggestion
        //   2. The amount of gas provided to the execution context of the the target contract is at
        //       least the gas limit specified by the user. If there is not enough gas in the
        //       callframe to accomplish this, `callWithMinGas` will revert.
```

Reply:
Added this with a slight modification, this is more clear than the current comment imo :100: